### PR TITLE
Improve UI/UX

### PR DESCRIPTION
**Pull request by choucl**
- Add blink effect when the user's turn started
- Add simple tutorial messages when game started

### CODE REVIEW

File: src/client.c [hunk 1]
```diff
@@ -398,18 +398,38 @@ int main(int argc, char *argv[])
...
+    
+    int first_in = 1;
     do {
         refresh();
 
         if (gcb->turn == 0) {
+            if (first_in) {
+                shift_msg(strs, colors);
+                snprintf(strs[6], MAX_LOG_LEN,
+                         "[Hint] It's your turn, press 'H' for more hint!");
```
[the remark this answers is not in the document]
Comment (choucl) [reply src/client.c:428]: Is it better for "placing hint"?
```suggestion
                         "[Hint] It's your turn, press 'H' for placing hint!");
```

File: src/client.c [hunk 2]
```diff
@@ -398,18 +398,38 @@ int main(int argc, char *argv[])
     rcb->coord.x = 0;
     rcb->coord.y = 0;
     
+
+    snprintf(strs[4], MAX_LOG_LEN,
+             "[Hint] Opponent paried! Game start!");
+    snprintf(strs[5], MAX_LOG_LEN,
+             "[Hint] ' ' for temp choice, 'q' for cancelling");
+    snprintf(strs[6], MAX_LOG_LEN,
+             "[Hint] 'c' for checking opponent's tiles");
```
[the remark this answers is not in the document]
Comment (choucl) [reply src/client.c:407]: Great!

[the remark this answers is not in the document]
Comment (choucl) [reply src/client.c:407]: But should we use upper case? Or lower case will be fine?

File: src/client.c [hunk 3]
```diff
@@ -398,18 +398,38 @@ int main(int argc, char *argv[])
     rcb->coord.x = 0;
     rcb->coord.y = 0;
     
+
+    snprintf(strs[4], MAX_LOG_LEN,
+             "[Hint] Opponent paried! Game start!");
+    snprintf(strs[5], MAX_LOG_LEN,
+             "[Hint] ' ' for temp choice, 'q' for cancelling");
```
[the remark this answers is not in the document]
Comment (choucl) [reply src/client.c:405]: Great, this is clearer

File: src/client.c (hunk 2 above)
[the remark this answers is not in the document]
Comment (yuyuranium) [reply src/client.c:407]: Do you mean the first character?

[the remark this answers is not in the document]
Comment (choucl) [reply src/client.c:407]: Right

File: src/client.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (yuyuranium) [reply src/client.c:428]: I think "hints" is clear enough but it's still okay to use "placing hint". BTW, "hint" is countable so you may append an 's' to it.

File: src/client.c (hunk 2 above)
[the remark this answers is not in the document]
Comment (yuyuranium) [reply src/client.c:407]: Yes, it should be in upper case. I didn't notice that all the other messages start with upper case.

**Pull request merged**